pkg/egressgateway/manager_privileged_test.go

```diff
@@ -593,8 +579,7 @@ func TestPrivilegedEgressGatewayManager(t *testing.T) {
 
 	// Update the endpoint labels in order for it to not be a match
 	_ = updateEndpointAndIdentity(&ep1, id1, map[string]string{})
-	addEndpoint(t, k.endpoints, &ep1)
-	waitForReconciliationRun(t, egressGatewayManager, reconciliationEventsCount)
+	reconciliationEventsCount = addEndpointAndReconcile(t, egressGatewayManager, reconciliationEventsCount, k.endpoints, &ep1)
```

Comment:
Nit: there are a number of ineffectual assignments due to the fact that `reconciliationEventsCount` is not used again in the current test (see check failures from linting).